changelog.md

```diff
@@ -33,9 +33,9 @@
   implemented. These are now implemented purely in ``system.nim`` without compiler
   support. There is a new "heterogenous" slice type ``system.HSlice`` that takes 2
   generic parameters which can be ``BackwardsIndex`` indices. ``BackwardsIndex`` is
-  produced by ``system.^``.
-  This means if you overload ``[]`` or ``[]=`` you need to ensure they also work
-  with ``system.BackwardsIndex`` (if applicable for the accessors).
+  produced by ``system.^``.  Using the ``BackwardsIndex`` on arrays
```

Comment:
This is unrelated to your PR.

Reply:
yes you are right. it was supposed to be in the other PR. Oops